### ISSUE TITLE
Heb Subs Premium

### ISSUE THREAD

**Issue by Nitzantomer1998** [danamag/stremio-addons-list#400]
### Addon Manifest URL

https://stremiohebsubs.onrender.com/manifest.json

### Addon Description

Stremio addon for Hebrew subtitles from all the top notch websites in one convenient location.
Currently support Wizdom, OpenSubtitles, Ktuvit & Machine Translation

### Language of Content

Hebrew (he-IL)

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [X] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by Aslinjean** [comment]
Hello team,

If I pay for an annual subscription, will I have access to watch movies/series on "Heb Subs" in my native language? Because I do not see the list of available languages on "Heb Subs". If you have a list of languages available on Heb Subs, please share it with me, as I see that Heb Subs supports Google Translate and Bing. In my opinion, it should also support French or Haitian Creole, but since I do not have access to check the available languages on Heb Subs, I thought it was important to write this message to find out if Heb Subs is really suitable for me or not. Thank you in advance for your response.

**Comment by Nitzantomer1998** [comment]
Hi there,

Heb Subs exclusively provides Hebrew subtitles, as implied by the name "Heb."

There are two versions of Hebrew subtitles available:
1. Human Translation: Sources include "Wizdom," "OpenSubtitles," and "Ktuvit."
2. Machine Translation: Powered by "Google Translate" and "Bing Translate."

**Comment by Aslinjean** [comment]
Hi there,

Heb Subs exclusively provides Hebrew subtitles, as implied by the name "Heb."

There are two versions of Hebrew subtitles available:

1. Human Translation: Sources include "Wizdom," "OpenSubtitles," and "Ktuvit."

2. Machine Translation: Powered by "Google Translate" and "Bing Translate."

If I understand correctly, if I pay for a subscription, I will not be able to choose the language I want, whether it's through Google Translate or Bing, which support almost all existing languages in the world?

**Comment by Aslinjean** [comment]
> Hi there,
> 
> 
> 
> Heb Subs exclusively provides Hebrew subtitles, as implied by the name "Heb."
> 
> 
> 
> There are two versions of Hebrew subtitles available:
> 
> 1. Human Translation: Sources include "Wizdom," "OpenSubtitles," and "Ktuvit."
> 
> 2. Machine Translation: Powered by "Google Translate" and "Bing Translate."

If I understand correctly, if I pay for a subscription, I will not be able to choose the language I want, whether it's through Google Translate or Bing, which support almost all existing languages in the world?

**Comment by Nitzantomer1998** [comment]
Correct, currently it's working by translating from English into Hebrew. The reason for this is that subtitles must exist before they can be translated, and English subtitles are typically the first to be released. Therefore, translating from English is the fastest way to obtain a Hebrew subtitle.

**Comment by Aslinjean** [comment]
> Correct, currently it's working by translating from English into Hebrew. The reason for this is that subtitles must exist before they can be translated, and English subtitles are typically the first to be released. Therefore, translating from English is the fastest way to obtain a Hebrew subtitle.

Okay, I understand. Anyway... I don't know if in the future you will make your add-on accessible to everyone and not just to your native language, but it would be a nice initiative because there are not many extensions on Stremio that offer this possibility. Anyway... Whatever happens, as soon as your extension supports other languages, many Stremio users will support you financially (myself included).

**Comment by Nitzantomer1998** [comment]
Okey, thank you for the feedback.
After i will have time (Currently student) i will might extend the addon to so do.

**Comment by Aslinjean** [comment]
> Okey, thank you for the feedback.
> 
> After i will have time (Currently student) i will might extend the addon to so do.

I would like to express my satisfaction with this announcement, which is excellent news both for me and for all Stremio users. Thank you for this initiative, and have a great day.

**Comment by hayekil** [comment]
How is this addon different from the Wizdon addon ?

**Comment by Nitzantomer1998** [comment]
Here are a few benefits:

1. **Multiple Subtitle Providers**: It combines multiple subtitle providers, including "Wizdom", "OpenSubtitles", and "Ktuvit", into one convenient addon. This means you get three extensions in one.

2. **Machine Translation**: It offers a "Machine Translation" service, currently supporting Google Translate and Bing Translate. This feature takes existing English subtitles and translates them into Hebrew for you.

3. **Support for Additional Subtitle Formats**: It supports more subtitle formats than Stremio, converting unsupported subtitles into formats that Stremio can handle.

4. **Fixing Punctuation**: sometimes subtitles punctuation are mixed up, the extension fix it.

5. **Error Resistance**: It has enhanced error resistance and provide well informed message for it, instead of making the annoying orange warning in Stremio.

There's more, but the biggest reason its because of the following:
**The addon is continuously evolving with time and user requests, ensuring it remains up-to-date and effective.**

**Comment by luizclaudio98** [comment]
Hello, I'm looking forward to an extension that groups subtitles for Brazilian Portuguese. The others don't work or show few options, making it very difficult for us to watch. Please include us in your project if possible! I will be eagerly following.
